restli-common/src/main/java/com/linkedin/restli/common/validation/RestLiDataValidator.java

```diff
@@ -286,15 +295,38 @@ public void validate(ValidatorContext context)
     {
       super.validate(context);
       DataElement element = context.dataElement();
-      if (_readOnlyRestrictedPredicate.evaluate(element))
+      System.out.println("Validate: " + element);
```

Comment:
Did you use this for debug? Not recommend to add it here.

Reply:
Good catch, lol